lib/tasks/enterprises.rake

```diff
@@ -1,6 +1,17 @@
 require 'csv'
 
 namespace :ofn do
+  desc 'deletes all orders and associated entities of an enterprise'
+  task :truncate_distributor, [:distributor_id] => :environment do |_task, args|
+    orders = Spree::Order.where(distributor_id: args.distributor_id)
+
+    inventory_units = Spree::InventoryUnit.where(order_id: orders.pluck(:id))
+    inventory_units.delete_all
```

Comment:
I know that `delete_all` is much faster but if someone decided to add a destroy callback to inventory units, they will probably not change this class.